dev/tools/bin/conductor

```diff
@@ -34,6 +34,7 @@ function follow_links() (
 
 PROG_NAME="$(follow_links "${BASH_SOURCE[0]}")"
 BIN_DIR="$(cd "${PROG_NAME%/*}" ; pwd -P)"
-DART_BIN="$BIN_DIR/../../../bin/dart"
+REPO_DIR="$BIN_DIR/../../.."
```

Comment:
This script is called by a LUCI recipe, so this has to remain until LUCI is updated to point to the new entrypoint